welcome/index.adoc

```diff
@@ -101,10 +101,8 @@ ifndef::openshift-origin[]
 
 - **xref:../installing/installing_ibm_z/preparing-to-install-on-ibm-z.adoc#preparing-to-install-on-ibm-z[Install a cluster on {ibm-z-name} and {ibm-linuxone-name}]**: On {ibm-z-name} and {ibm-linuxone-name}, you can install {product-title} on user-provisioned infrastructure.
 endif::openshift-origin[]
-////
-// Addition will be tracked through https://issues.redhat.com/browse/OSDOCS-9514
-- ** Install a cluster on {oci-first}**: You can use the Assisted Installer or the Agent-based Installer to install a cluster on {oci}. This means that you can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments.
-////
+
+- **Install a cluster on {oci-first}**: You can use the Assisted Installer or the Agent-based Installer to install a cluster on {oci}. This means that you can run cluster workloads on infrastructure that supports dedicated, hybrid, public, and multiple cloud environments. See xref:../installing/installing_oci/installing-oci-assisted-installer.adoc[Using the Assisted Installer to install a cluster on {oci}] and xref:../installing/installing_oci/installing-oci-agent-based-installer.adoc[Using the Agent-based Installer to install a cluster on {oci}].
```

Comment:
Not sure if the exact title of the second link needs to match the actual doc title but they're slightly different. I'll go ahead and merge and you can create a new PR if should be changed.